src/Controller/PluginFamily.php

```diff
@@ -12,7 +12,7 @@ class PluginFamily implements PluginFamilyInterface {
 	 *
 	 * @var string
 	 */
-	private $version = '1.0.6';
+	private $version = '1.0.7';
```

Comment:
```suggestion
	private $version = '1.0.8';
```
We released 1.0.7 previously without changing this version number.